scripts/test/Muon/load_run_widget/loadrun_presenter_current_run_test.py

```diff
@@ -0,0 +1,151 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+import sys
+from Muon.GUI.Common.load_run_widget.model import LoadRunWidgetModel
+from Muon.GUI.Common.load_run_widget.view import LoadRunWidgetView
+from Muon.GUI.Common.load_run_widget.presenter import LoadRunWidgetPresenter
+
+from Muon.GUI.Common.muon_load_data import MuonLoadData
+import Muon.GUI.Common.utilities.muon_file_utils as fileUtils
+
+import unittest
+
+if sys.version_info.major == 3:
+    from unittest import mock
+else:
+    import mock
+
+from PyQt4 import QtGui
+from PyQt4.QtGui import QApplication
+
+# global QApplication (get errors if > 1 instance in the code)
+QT_APP = QApplication([])
```

Comment:
I think for this you could use https://github.com/mantidproject/mantid/blob/master/scripts/Muon/GUI/Common/mock_widget.py as seen in https://github.com/mantidproject/mantid/blob/master/scripts/test/Muon/transformWidget_test.py#L27. Unless I have missed something